_data/packages.yml

```diff
@@ -288,18 +300,19 @@
   issue_link: https://github.com/pyOpenSci/software-submission/issues/147
   joss:
   partners:
-    - sunpy
```

Comment:
Same thing here, we had made it a strict rule for only the partners in that enum in parsing, but it looks like some people have put in custom partners. In the future stuff like this will be easier with pyos bot where we can be like "hey there is an unexpected partner here, if ya want to add it just PR an additional option in this enum, but otherwise it wont be parsed" in the issue at the time its added, but ig we just have to make these calls ad hoc in retrospect

Reply:
yes - definitely. This one is actually my fault :) sunpy is a potential future partner but we haven't formalized it yet. pinging @nabobalis ! 😸 

At some point we should circle back on whether we do want to formalize a partnership!! No pressure and no rush.  ✨ but @sneakers-the-rat for now let's remove ensure that the bot is following the Literal values provided in the model rather than allowing spurious ones. another great catch

Reply:
Hi @lwasser, there is from my side but right now sunpy is undergoing a new ogranzation process, electing a steering committee. Hopefully when that is finished. The new people in charge can swing back to this. 

I personally want to go full steam ahead with this but I lack the authority to speak for the sunpy project. The new steering committee will.

Reply:
hi @nabobalis no problem! i fully understand. when the time is right let's connect / or connect me with the proper folks to discuss this further. there is no rush! for the time being let's then plan to not include sunpy as a formal partnership even tho we did review the core package!! thank you so much for the speedy reply!!

Reply:
Hello @lwasser, sorry for bring up a comment on an older PR but the steering committee for sunpy has been selected.

They are @cadair, @hayesla and @ayshih 

Hopefully all them will get in touch with you soon about taking this forward.